Packages/MIES/MIES_AnalysisBrowser.ipf

```diff
@@ -2649,19 +2649,26 @@ static Function AB_AddExperimentEntries(string win, WAVE/T entries)
 			symbPath = GetUniqueSymbolicPath()
 			NewPath/O/Q/Z $symbPath, entry
 			if(GetCheckBoxState(win, "check_load_pxp"))
-				pxpList = GetAllFilesRecursivelyFromPathAsList(symbPath, extension = ".pxp")
-				uxpList = GetAllFilesRecursivelyFromPathAsList(symbPath, extension = ".uxp")
-			else
-				pxpList = ""
-				uxpList = ""
+				WAVE/Z/T pxps = GetAllFilesRecursivelyFromPath(symbPath, regex = "\.(pxp|uxp)$")
 			endif
 			if(GetCheckBoxState(win, "check_load_nwb"))
-				nwbList = GetAllFilesRecursivelyFromPathAsList(symbPath, extension = ".nwb")
-			else
-				nwbList = ""
+				WAVE/Z/T nwbs = GetAllFilesRecursivelyFromPath(symbPath, regex = "\.nwb$")
 			endif
 			KillPath/Z $symbPath
-			WAVE/T fileList = ListToTextWave(SortList(pxpList + uxpList + nwbList, FILE_LIST_SEP), FILE_LIST_SEP)
+
+			if(!WaveExists(pxps) && !WaveExists(nwbs))
+				break
+			endif
+
+			if(WaveExists(pxps))
+				WAVE/T fileList = pxps
+			elseif(WaveExists(nwbs))
+				WAVE/T fileList = nwbs
+			else
+				Concatenate/NP=(ROWS)/FREE/T {nwbs, pxps}, fileList
```

Comment:
This is never reached, but should be reached for 
`WaveExists(pxp) && WaveExists(nwb)`, so move it to the start of the if else chain.